reco_utils/recommender/deeprec/deeprec_utils.py

```diff
@@ -539,11 +555,27 @@ def cal_metric(labels, preds, metrics):
         elif metric == "mean_mrr":
             mean_mrr = np.mean(mrr_score(labels, preds))
             res["mean_mrr"] = round(mean_mrr, 4)
-        elif metric == "ndcg":
-            ndcg_list = [2, 4, 6, 8, 10]
-            for r in ndcg_list:
-                ndcg_temp = np.mean(ndcg_score(labels, preds))
-                res["ndcg{0}".format(r)] = round(ndcg_temp, 4)
+        elif metric.startswith("ndcg"):  # format like:  ndcg@2;4;6;8
+            ndcg_list = [1, 2]
```

Comment:
If the metric is `ndcg` it will be equivalent to `ndcg@1;2`. Is that correct? Is this what we expect?

Reply:
yes, in case of user didn't assign the k value, we use [1,2] as the defualt